modules/installing-ocp-agent.adoc

```diff
@@ -94,16 +94,18 @@ sshKey: |
     name: sno-cluster
   rendezvousIP: 192.168.111.80 <1>
   hosts: <2>
-    - hostname: master-0
+    - hostname: master-0 <3>
       interfaces:
         - name: eno1
           macAddress: 00:ef:44:21:e6:a5
-      networkConfig:
+      networkConfig: <4>
         interfaces:
           - name: eno1
             type: ethernet
             state: up
             mac-address: 00:ef:44:21:e6:a5
+          rootDeviceHints: <5>
```

Comment:
This is indented at the wrong level, it should be parallel with hostname/interfaces/networkConfig (and be moved up or down so it doesn't appear in the middle of the network config.